test/apps/scripts/network.sh

```diff
@@ -22,6 +22,7 @@ echo "Start network test......"
 ./http_server &
 ./http_client
 ./tcp_err
+./tcp_poll_err
```

Comment:
I think we can call this file `tcp_poll` now, since it is not really related to the error codes. Right?